test/common.js

```diff
@@ -142,7 +147,20 @@ exports.validateContent = function validateContent(data, t, options) {
   }
 
   // Check report System Information section
-  const sysInfoSection = getSection(reportContents, 'System Information');
+  // If the report contains a glibc version, check it against libc.so.6
+  const glibcMatch = glibcRE.exec(nodeReportSection);
+  if (glibcMatch != null) {
+    const libcMatch = libcRE.exec(sysInfoSection);
+    if (libcMatch != null) {
+      const libcPath = libcMatch[1];
+      const child = child_process.spawnSync(libcPath, { encoding: 'utf8' });
```

Comment:
Huh, I see glibc has a `__libc_main` entry point in its ELF header but that only works when the .so has the executable bit set.  I don't know if that is something you can rely on but it is the case on the three Linux systems (Debian, Ubuntu, Fedora) in my household.